### PR TITLE
feat: Rich config

### PR DESCRIPTION
**Pull request by ukstv**
Includes block subgraph and skew.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -32,19 +28,24 @@ const ceramic = new Ceramic() // connects to localhost:7007 by default
...
     'eip155:1': {
-      // Asset namespace
-      erc721: 'https://api.thegraph.com/subgraphs/name/xxx/yyy',
-      // erc721: 'http://localhost:8000/subgraphs/name/aoeu/qjkx' // also works!
-      erc1155: 'https://api.thegraph.com/subgraphs/name/abc/xyz'
+      blocks: "https://api.thegraph.com/subgraphs/name/yyong1010/ethereumblocks",
+      skew: 15000,
+      assets: {
+        erc721: "https://api.thegraph.com/subgraphs/name/sunguru98/mainnet-erc721-subgraph",
+        erc1155: "https://api.thegraph.com/subgraphs/name/sunguru98/mainnet-erc1155-subgraph",
```
Comment (oed) [inline README.md:37]: Is it possible to provide only one of the two standards? Certain chains (e.g. Palm) only uses 1155.

File: README.md [hunk 2]
```diff
@@ -32,19 +28,24 @@ const ceramic = new Ceramic() // connects to localhost:7007 by default
...
+      },
+    },
+    'eip155:4': {
```
Comment (oed) [inline README.md:40]: Can we say something about what a CAIP-2 chainId is somewhere so that people can learn what this is?

**Review by oed — APPROVED** [review]
lgtm

File: README.md (hunk 1 above)
Comment (ukstv) [reply README.md:37]: It is indeed.

**Pull request merged**